ui/raidboss/data/06-ew/raid/p10s.ts

```diff
@@ -575,14 +575,37 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'P10S Jade Passage',
-      type: 'StartsUsing',
-      netRegex: { id: '828C', capture: false },
+      // Track addition of Arcane Sphere combatants
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '12356' },
       suppressSeconds: 5,
-      infoText: (_data, _matches, output) => output.text!(),
+      infoText: (_data, matches, output) => {
+        const y = parseInt(matches.y);
+        const safeSpot: { [y: number]: string } = {
+          86: output.boxes!(),
```

Comment:
Can this be simplified to be `Math.floor(y / 2) % 2 == 1` sorta thing?

Reply:
Yea, that works.